code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -64,18 +64,19 @@
 // Syndie survival box
 /obj/item/storage/box/survival/syndie
 	name = "operation-ready survival box"
-	desc = "A box with the essentials of your operation. This one is labelled to contain an extended-capacity tank."
+	desc = "A box with the essentials of your operation. This one is labelled to contain an extended-capacity tank and a handy guide on survival."
 	icon_state = "syndiebox"
 	illustration = "extendedtank"
 	mask_type = /obj/item/clothing/mask/gas/syndicate
 	internal_type = /obj/item/tank/internals/emergency_oxygen/engi
-	medipen_type = null
+	medipen_type =  /obj/item/reagent_containers/hypospray/medipen/atropine
 
 /obj/item/storage/box/survival/syndie/PopulateContents()
 	..()
 	new /obj/item/crowbar/red(src)
 	new /obj/item/screwdriver/red(src)
 	new /obj/item/weldingtool/mini(src)
+	new /obj/item/paper/fluff/operative(src)
```

Comment:
So people know how I'm fixing this, while #71440  will resolve this problem, I'm putting the tools into a parcel you tear open. It will be small sized. This helps free up their box space overall.